products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -668,6 +668,17 @@ public final class BuildSystem {
         }
     }
 
+    /// Build a specific target.
+    ///
+    /// The client is responsible for ensuring only one build is ever executing concurrently.
+    ///
+    /// - parameter target: Name of the target to build.
+    /// - returns: True if the build was successful, false otherwise.
+    public func build(target: String) -> Bool {
```

Comment:
Maybe this should be flipped with the node variant?

```swift
func build(target: String? = nil) -> Bool
func build(node: String) -> Bool
```

Reply:
Yep, sounds logical